gadgets/trace_malloc/program.bpf.c

```diff
@@ -11,13 +11,14 @@ struct event {
 	__u32 pid;
 	__u64 addr;
 	bool is_malloc;
+	__u8 comm[TASK_COMM_LEN];
```

Comment:
gadget.yaml will need to be updated. This can be done automatically with `ig image build --update-metadata`